.tests/integration/config.yaml

```diff
@@ -26,6 +26,15 @@ cnvkit_scatter:
 cnvkit_vcf:
   container: "docker://hydragenetics/cnvkit:0.9.9"
 
+cnvkit_seg:
+  container: "docker://marrip/cnvkit:0.9.9"
+
+purecn:
+  container: "docker://markusriester/purecn:2.2.0"
+
+purecn_coverage:
+  container: "docker://markusriester/purecn:2.2.0"
+
```

Comment:
Add required options in config and dummy files in .tests/integration
          - genome
          - segmentation_method
          - interval_padding